test/blog/articles.py

```diff
@@ -0,0 +1,20 @@
+import os
+import unittest
+
+from bin.fixtures import create_article
+from blog.models import Article
+
+
+class ArticleTest(unittest.TestCase):
```

Comment:
This class [already exists](https://github.com/neomad-team/neomad.org/blob/master/blog/test.py#L6)
You should append your method to that class.